### DISCUSSION TITLE
What does NCLX stand for? Are the enums different from CICP?

### DISCUSSION THREAD

**Discussion by gregbenz** [AOMediaCodec/libavif#2487] [Q&A]
I've seen several references to NCLX, but never found a definition of what the acronym actually stands for or how it differs from CICP.

Does NCLX have its own enums, or are CICP ("code independent code point") values just used in NCLX?
Are these values specific to a format like AVIF, or would the same enums/terms apply to other formats like JPEG XL?

For my own education, I'd appreciate any clarity or pointers to reference/standards docs which might clarify these values. I run into bugs involving them (stripped / altered color profiles), and it would help for me to better understand the mechanics involved.

**Comment by wantehchang** [comment]
Greg: I know nclx comes from QuickTime. A quick Web search for "nclx QuickTime" gives this: https://poynton.ca/notes/misc/sde-nclc-vui-nclx.html
